app/views/activities/_feed_entry.html.slim

```diff
@@ -1,10 +1,14 @@
 li class="activity #{activity.kind}"
   = link_to activity.source_url
-    = image_tag(activity.img_url || '/images/screenshot-missing.png')
-  = content_tag :h4, link_to(activity.title, activity.source_url, class: 'title')
+  	- if activity.img_url.present?
```

Comment:
slim / haml and the like are most picky when it comes to indentation. I can only think of spaces not adding up somewhere.